docs/usage.md

```diff
@@ -114,25 +95,19 @@ This version number will be logged in reports when you run the pipeline, so that
 
 To further assist in reproducbility, you can use share and re-use [parameter files](#running-the-pipeline) to repeat pipeline runs with the same settings without having to write out a command with every single parameter.
```

Comment:
In the "Updating the pipeline" section there is a reference to the command `nextflow pull nf-core/staramr` to update. Can you change this to `nextflow pull phac-nml/staramrnf`?

Reply:
Yes! 5ea372a93593dc103ff5ef9593d0b10eee580014